apps/music/js/app.js

```diff
@@ -44,18 +44,6 @@ if (navigator.mozSetMessageHandler) {
   navigator.mozSetMessageHandler('activity', activity => onActivity(activity));
 }
 
-var unknownArtist;
-var unknownTitle;
```

Comment:
My understanding is that the reason why there were these two globally scoped variables was performance.  They were used in `scanProgress` which I'm not sure when is called, but it sounds like it might be a listener to `progress` events :)  @justindarc, how can I test this?  Do I need to load a lot of mp3's on my sd card?